### PR TITLE
extract static PebbleUtils from WebTemplateToadlet for easier reuse

### PR DESCRIPTION
**Pull request by ArneBab**
This is refactoring via IntelliJ. It compiles and I hope it works …

### CODE REVIEW

**Comment by Bombe** [comment]
ENOTESTS, rejected! 😄

**Comment by Bombe** [comment]
Wait, are you using hyphanet’s repositories for your own development stuff? I was looking for this branch in your repository but was unable to find it…

**Comment by Bombe** [comment]
I put some tests in https://github.com/Bombe/fred/tree/extract-pebble-utils-tests.

**Comment by ArneBab** [comment]
You’re awesome! Thank you! (I use branches here, yes — that way someone else can take over development of the PR. I’ve seen too many stale PRs that I’d have had to re-create to continue the work)

File: test/freenet/clients/http/utils/L10nExtensionTest.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	public void l10nFunctionRetrievesGivenArgumentWithPrefixFromContextToNodeL10n() {
+		Map<String, Object> arguments = singletonMap("0", "l10nFunctionTest");
+		EvaluationContext context = createEvaluationContext();
+		assertThat(l10nFunction.execute(arguments, null, context, 0), equalTo("Localized Value"));
+	}
+
+	private static EvaluationContext createEvaluationContext() {
+		return new EvaluationContext() {
+			@Override
+			public boolean isStrictVariables() {
```
Comment (ArneBab) [inline test/freenet/clients/http/utils/L10nExtensionTest.java:44]: should we enable strictVariables to have some error checking? (that does compile-time checks of the templates)

File: build.gradle [hunk 2]
```diff
@@ -172,6 +172,7 @@ task copyTestResourcesToClasses2 {
             include 'freenet/client/filter/*/**'
             include 'freenet/crypt/ciphers/rijndael-gladman-test-data/**'
             include 'freenet/l10n/*properties'
+            include 'freenet/clients/http/templates/**'
```
Comment (ArneBab) [inline build.gradle:175]: :heart:

File: test/freenet/clients/http/utils/L10nExtensionTest.java (hunk 1 above)
Comment (Bombe) [reply test/freenet/clients/http/utils/L10nExtensionTest.java:44]: I don’t think so. This context is only used in the test, and we don’t need to test that Pebble handles its contexts correctly. 🙂

Comment (ArneBab) [reply test/freenet/clients/http/utils/L10nExtensionTest.java:44]: Ah, yes, that’s sensible. Thanks!

**Comment by ArneBab** [comment]
> > You’re awesome! Thank you! (I use branches here, yes — that way someone else can take over development of the PR. I’ve seen too many stale PRs that I’d have had to re-create to continue the work)
> 
> Yeah but can you please use _your own_ repository instead of the project’s? smile

OK, I will :)

**Review by Bombe — APPROVED** [review]

**Pull request merged**